index.bs

```diff
@@ -1520,94 +1459,189 @@ If the attestation root certificate is not dedicated to a single WebAuthn Authen
-    - Verify that {{PackedAttestation/x5c}} meets the requirements in [[#packed-attestation-cert-requirements]].
+    If DAA is in use, the authenticator produces |sig| by following the procedure in [[#authenticator-signature]] with the given
+    <a>clientDataHash</a> and <a>authenticatorData</a> as inputs, using DAA-Sign with a DAA root key selected through an
+    authenticator-specific mechanism (see [[!FIDOEcdaaAlgorithm]]). It sets |alg| to the algorithm of the DAA root key and
+    |daaKey| to the DAA root key.
 
-    If {{PackedAttestation/daaKey}} is present, then the attestation type is DAA. In this case:
-    - Verify that {{PackedAttestation/alg}} is "ED256" or "ED512".
-    - Perform DAA-Verify on {{PackedAttestation/signature}} (see [[!FIDOEcdaaAlgorithm]]).
-    - If {{PackedAttestation/x5c}} contains an extension with OID `1 3 6 1 4 1 45724 1 1 4` (id-fido-gen-ce-aaguid) verify that
-        the value of this extension matches the AAGUID in the {{AuthenticationAttestation/authenticatorData}}.
+    If self attestation is in use, the authenticator produces |sig| by following the procedure in
+    [[#authenticator-signature]] with the given <a>clientDataHash</a> and <a>authenticatorData</a> as inputs, using the
+    credential private key. It sets |alg| to the algorithm of the credential private key, and omits the other fields.
 
-    If neither {{PackedAttestation/x5c}} nor {{PackedAttestation/daaKey}} is present, self attestation is in use.
-    - Verify the signature using the <a>credential public key</a>.
-    - Validate that {{PackedAttestation/alg}} matches the algorithm in {{AuthenticationAttestation/authenticatorData}}.
+: Verification procedure
+:: If both |x5c| and |daaKey| are present, terminate this procedure with an error.
+
+    If |x5c| is present, this indicates that the attestation type is not DAA. In this case:
+    - Follow the procedure in [[#authenticator-signature-verification]] to verify that |sig| is a valid signature over the given
+        <a>authenticatorData</a> and <a>clientDataHash</a>  using the attestation public key in |x5c| with the algorithm
+        specified in |alg|.
+    - Verify that |x5c| meets the requirements in [[#packed-attestation-cert-requirements]].
+    - If |x5c| contains an extension with OID `1 3 6 1 4 1 45724 1 1 4` (id-fido-gen-ce-aaguid) verify that
+        the value of this extension matches the AAGUID in the claimed <a>authenticatorData</a>.
+    - If successful, return attestation type Basic and trust path |x5c|.
+
+    If |daaKey| is present, then the attestation type is DAA. In this case:
+    - Verify that |alg| is "ED256" or "ED512".
+    - Follow the procedure in [[#authenticator-signature-verification]] to verify that |sig| is a valid signature over the given
+        <a>authenticatorData</a> and <a>clientDataHash</a>  using DAA-Verify with |daaKey| (see [[!FIDOEcdaaAlgorithm]]).
```

Comment:
in this and all the other `verification procedure` subsections below, need to `s/clientDataHash/clientDataJSON/` because that is what is input to {{#authenticator-signature-verification}}